provider/apiutils/apiutils.go

```diff
@@ -109,6 +110,14 @@ func GetClassLabel(o api.Object) (string, bool) {
 	return class, found
 }
 
+func SetMachineNamespaceLabel(o api.Object, namespace string) {
+	metautils.SetLabel(o, ironcorev1alpha1.MachineNamespaceLabel, namespace)
+}
+
+func SetMachineNameLabel(o api.Object, namespace string) {
+	metautils.SetLabel(o, ironcorev1alpha1.MachineNameLabel, namespace)
```

Comment:
are you sure with namespace variable?

Reply:
I will change it to `name`. Thanks for pointing.

Reply:
Changed.